MatrixSDK/MXSession.h

```diff
@@ -427,6 +430,9 @@ typedef void (^MXOnBackgroundSyncFail)(NSError *error);
                     visibility:(MXRoomDirectoryVisibility)visibility
```

Comment:
Is is possible to create a new method rather than making the existing one complicate ?
It breaks a lot of things for no added value most of the time. We already have 2 "createRoom", we can have 3.